output/exceptions.py

```diff
@@ -0,0 +1,31 @@
+from output.status import Status
+
+class MissingInput(Exception):
+    def __init__(self, status = Status.MissingInput, message = "there is a missed required input"):
+        self.status = status
+        self.message = message
```

Comment:
You don't need self.message here. The parent class has a message as an instance variable.

Reply:
You need to read about super in Java. Read about inheritance and why we use super in constructor